tech-specs/src/explore/design/dkg.md

```diff
@@ -1 +1,178 @@
-# Distributed key generation gossip
+# Distributed key generation
+
+Here we give an overview of how the DKG protocol works with the ledger and 
+the roles of each node in the network per this protocol. This portion of the 
+book focuses solely on the production of a new public encryption key via the
+DKG scheme. It does not cover encryption, decryption, or validation of transactions.
+
+Some parts of the DKG protocol may be done through the gossip layer or through
+off-chain communication. However, the entire DKG cannot exist solely in the 
+gossip layer as consensus on the final shared secrets must be achieved. This 
+will require posting information on chain.
+
+## Starting the Protocol
+
+The protocol to generate a new key will be scheduled to begin at specific 
+block heights, in pratices this will be at the beginning of every eposh.
+This way all the validators know when to begin the announcement phase. In 
+addition to starting at the pre-determined blocks heights, if the protocol 
+fails, all validators know to immediately start a new round of the protocol.
+
+It is important to note that the key that is being generated is to be used
+encrypting transactions in the epoch following the current one. However,
+this relies on the assumption that after the validators sets change between
+epochs that there is still validators whose combined weight exceeds 
+\\( \frac{2}{3} \\) of \\W\\. Otherwise, it will be impossible to decrypt
```

Comment:
why? clarify please

Reply:
This is because the "keys for decryption" were only given to those who participated in the previous epoch. Tried to clarify that.